docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+  version: "3"
+
+  services:
+    db:
+      image: "postgres:12"
+      restart: unless-stopped
+      networks:
+        - "gpsnet"
+      env_file:
+        - ".env"
+      volumes:
+        - ./db-data:/var/lib/postgresql/data
+        - ./migrator/init:/docker-entrypoint-initdb.d/
+      ports:
+        - 5432:5432
+
+    migrator:
```

Comment:
What do you think about using a command for migrations?
https://github.com/lvl484/positioning-filter/pull/7/files#diff-04c6e90faac2675aa89e2176d2eec7d8R70

Reply:
It  will not allow to us make dockers automation containers up for us in future with others services